docs/checkpassword.rst

```diff
@@ -11,3 +11,14 @@ To enable this feature:
 .. code-block:: php
 
     $use_checkpassword = true;
+
+Check password history
+----------------------
+
+The password history can be checked by enabling this feature.
```

Comment:
```suggestion
When verifying the password, Service Desk can parse the password history to check if the current is part of it. This can be useful to tell a user that the password was working before but has been changed since.
```